exec/java-exec/src/main/java/org/apache/drill/exec/store/AbstractSchema.java

```diff
@@ -231,4 +231,21 @@ public void dropTable(String tableName) {
     }
     return tables;
   }
-}
\ No newline at end of file
+
+  public List<Pair<String, Schema.TableType>> getTableNamesAndTypes(boolean bulkLoad, int bulkSize) {
+    final List<String> tableNames = Lists.newArrayList(getTableNames());
+    final List<Pair<String, Schema.TableType>> tableNamesAndTypes = Lists.newArrayList();
+    final List<Pair<String, ? extends Table>> tables;
+    if (bulkLoad) {
+      tables = getTablesByNamesByBulkLoad(tableNames, bulkSize);
```

Comment:
why do we even have this option to do bulkLoad or not ? why not just do bulkLoad always ?